src/main/java/org/gridsuite/modification/server/elasticsearch/EquipmentInfosService.java

```diff
@@ -19,9 +21,23 @@
  */
 @Service
 public interface EquipmentInfosService {
-    EquipmentInfos add(@NonNull EquipmentInfos equipmentInfos);
+    EquipmentInfos addEquipmentInfos(@NonNull EquipmentInfos equipmentInfos);
 
-    void delete(@NonNull String equipmentId, @NonNull UUID networkUuid);
+    TombstonedEquipmentInfos addTombstonedEquipmentInfos(@NonNull TombstonedEquipmentInfos tombstonedEquipmentInfos);
 
-    Iterable<EquipmentInfos> findAll(@NonNull UUID networkUuid); // Only for tests
+    void addAllEquipmentInfos(@NonNull final List<EquipmentInfos> equipmentsInfos);
+
+    void addAllTombstonedEquipmentInfos(@NonNull final List<TombstonedEquipmentInfos> tombstonedEquipmentsInfos);
+
+    void deleteEquipmentInfos(@NonNull String equipmentId, @NonNull UUID networkUuid, @NonNull String variantId);
+
+    void deleteVariants(@NonNull UUID networkUuid, List<String> variantIds);
```

Comment:
Why a list ?

Reply:
We surely will have to remove multiple variants (when they will be invalidated for example)